src/cls/rbd/cls_rbd.cc

```diff
@@ -134,6 +134,9 @@ cls_method_handle_t h_mirror_image_status_get;
 cls_method_handle_t h_mirror_image_status_list;
 cls_method_handle_t h_mirror_image_status_get_summary;
 cls_method_handle_t h_mirror_image_status_remove_down;
+cls_method_handle_t h_mirror_image_create_marker_get;
```

Comment:
I don't think new cls methods are necessary -- we just need to adjust the rbd::mirror::image_replayer::BootstrapRequest state machine and add a new state to librbd::journal::MirrorPeerState to indicate the image is being "created".